hack/tests/origin-conformance.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash -ex
+
+PATH=`go env GOPATH`/bin::$PATH
+TEST_SUITE="${TEST_SUITE:-openshift/conformance/parallel/minimal}"
+
+cleanup() {
+  set +e
+
+  generate_artifacts
+
+  delete
+}
+
+fetch_origin() {
+  local branch="$1"
+  local remote="https://github.com/openshift/origin.git"
+
+  test -d /tmp/origin || git clone --depth=1 --branch=${branch} $remote /tmp/origin
+}
+
+install_binaries() {
+  # install ginkgo
+  go get -v github.com/onsi/ginkgo/ginkgo
+
+  # build extended.tests (regular user can't write to /usr/local)
+  # TODO: This will need updating once we move to 4.x
+  make -C /tmp/origin build-extended-test && mv -v /tmp/origin/_output/local/bin/linux/`go env GOARCH`/* `go env GOPATH`/bin/
+}
+
+function run-tests() {
```

Comment:
can we rename this to use snake case? 
```suggestion
function run_tests() {
```

Reply:
this is bash though, we don't do snake case in bash afaics

Reply:
I was referring to how the other functions in the script are named. This is the only one using a hyphen. Others use an underscore. Just a cosmetic thing...

Reply:
> we don't do snake case in bash afaics

Every other subroutine in this file uses snake case :smile: